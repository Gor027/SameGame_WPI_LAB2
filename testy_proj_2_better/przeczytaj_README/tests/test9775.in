}   S  S}}  S  } }  }}   }S }SS  }}S}}S  S} S }
}}SS}S}}} S S}   }S  } }} }S   SS   S}}}}}SS  S S  SSSS} } }}} }}}S}S
 S S}
 } S} S} }S}} S }}S SS
 }}S}}} }S  S} }  }}}}S} }}   S}}   S}  S }S S 
S}S} }  SS}}  S}} S S }S} S} SS }S}} SS } S}}} SS   }S    S}S}SS
}}SS} }SS S }}}S }} SS}S}   SSS  } S}S   S}}}SS}}}S} }}S}} }} SS} SSSS S}}
 S S}S}   S S}}  }    S  S S}S }SS}SS SS}S }}}SS SS  }} }}  S S}   S}}    
SS}} }}S}} SS S S} }}} }S}} SS SS}} } SS}  } SS   S }S}S
  }       }  }S}} SS  S}SS S}S}SS  }} S}}}SS S} } SS}S}} 
S}} S}  
S}S} }SSS} 
SS
S}}} }SS    }} S }S S}S} }S SSSS}  SSS}
 S}S}S}S }SS S} S}}}}}  }
