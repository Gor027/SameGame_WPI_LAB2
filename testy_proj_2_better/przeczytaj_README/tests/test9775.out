 } S
 S S
 }SSSS}
 }}S}}SS    S    }S }}              }
 }S}}}}}}}S S}S  }S S} }     S  S   S S    S
 }SS}S}}}}S }}S} }} }SS}}}S }SSS}  SS}} } }S S}      S
 S}S}}}S}S} S}}}}}S S}}S}}} SSS}}} }}}S SSS} }S      SS
}S}}}}}SSSS}}}}}}}}}SS}S}S}S}SS}SS}SS}S}}}}}SSS S S S}}}    }  }
}}}}}}SSS}S}S}SSS}S}}}S}S}S}SSSSS}}S}SSS}SS}}}SSS SS}}}S } }S SS   }S
S}}}S}SS}}}SS}S}}}SSS}S}SS}S}S}SSS}}}SSS}S}}}SS}}}S}}S}S}} }}}SS} SSSS
}}}S}S}SS}SS}S}SS}}}}}S}}}SSSSSS}}}SSS}}}}SS}S}S}SSS}S}}}}S}S}S}}}SS}}S}}
